frontend/src/layout/navigation/SideBar/SideBar.tsx

```diff
@@ -84,184 +84,194 @@ function Pages(): JSX.Element {
-                                                    onClick={() => setArePinnedDashboardsShown(false)}
-                                                    to={urls.dashboard(dashboard.id)}
-                                                />
-                                            ))
-                                        ) : (
-                                            <>
-                                                <div className="mb-2 flex items-center gap-2">
-                                                    <IconPin className="text-2xl text-muted-alt" />
-                                                    <span>
-                                                        <Link
-                                                            onClick={() => setArePinnedDashboardsShown(false)}
-                                                            to={urls.dashboards()}
-                                                        >
-                                                            Pin some dashboards
-                                                        </Link>
-                                                        <br />
-                                                        for them to show up here
-                                                    </span>
-                                                </div>
-                                            </>
-                                        )}
-                                    </div>
-                                ),
-                            },
-                        }}
-                    />
-                    <PageButton
-                        icon={<IconBarChart />}
-                        identifier={Scene.SavedInsights}
-                        to={urls.savedInsights()}
-                        sideAction={{
-                            icon: <IconPlus />,
-                            to: urls.insightNew(),
-                            tooltip: 'New insight',
-                            identifier: Scene.Insight,
-                            onClick: hideSideBarMobile,
-                        }}
-                    />
-                    <PageButton
-                        icon={<IconRecording />}
-                        identifier={Scene.SessionRecordings}
-                        to={urls.sessionRecordings()}
-                    />
-                    <PageButton icon={<IconFlag />} identifier={Scene.FeatureFlags} to={urls.featureFlags()} />
-                    {(hasAvailableFeature(AvailableFeature.EXPERIMENTATION) ||
-                        !preflight?.instance_preferences?.disable_paid_fs) && (
-                        <PageButton icon={<IconExperiment />} identifier={Scene.Experiments} to={urls.experiments()} />
-                    )}
-                    {featureFlags[FEATURE_FLAGS.WEB_PERFORMANCE] && (
+                    <div className="Sidebar__buttons">
```

Comment:
This entire diff is just wrapping the buttons in this component

Reply:
Any reason we don't markup the navigation as a list? That would give us a common container for each item in the navigation and would simplify the CSS needed to add the 1px spacing.

Reply:
No particular reason other than it not bringing any benefits that I am aware of?

Reply:
Swapped it over to ul/li combo (comes with the downside of having more CSS to counteract the defaults). At some point I'll look at if we can remove the default styles as they are almost never useful, just need to make sure they're not relied on anywhere.